### ISSUE TITLE
eslint cannot parse the rules

### ISSUE THREAD

**Issue by liemdo** [benmosher/eslint-plugin-import#932]
Hi,

I have the rules defined in the .eslintrc.yml like the following
```yml
...
  import/order:
  - 2
  - groups:
    - builtin
    - external
    - internal
    - - parent
      - sibling
      - index
    newlines-between: always-and-inside-groups
...
```

but I eslint complains that the configuration is invalid

> Configuration for rule "import/order" is invalid:
>       Value "always-and-inside-groups" must be an enum value.

the `newlines-between` option works with everything else except always-and-inside-groups

**Comment by liemdo** [comment]
I fixed this after upgrading eslint to the latest version.
